arkouda/series.py

```diff
@@ -1,10 +1,14 @@
+from __future__ import annotations
+
 from typing import List, Optional, Tuple, Union
 from warnings import warn
 
 import numpy as np  # type: ignore
 import pandas as pd  # type: ignore
 from pandas._config import get_option  # type: ignore
 from typeguard import typechecked
+
+import arkouda.dataframe
```

Comment:
if you change this to 
```python
from arkouda.dataframe import DataFrame
```
you won't need to do `arkouda.dataframe.DataFrame` and you'll be able to do just `DataFrame` instead

Reply:
@pierce314159 this was done due to an issue with the circular import that using `from` on the import causes. I know it is not ideal, but `import arkouda.dataframe` is more flexible.

Reply:
ah i see! i'm cool with this approach